chia/full_node/full_node_api.py

```diff
@@ -494,6 +494,7 @@ async def respond_signage_point(
                     f"{self.full_node.constants.NUM_SPS_SUB_SLOT}: "
                     f"{request.challenge_chain_vdf.output.get_hash()} "
                 )
+                self.full_node.signage_point_times[request.index_from_challenge] = time.time()
```

Comment:
Generally, `time.time()` isn't ideal because it's a "hidden global" that can make testing non-deterministic by making this function use hidden state (the system clock). Consider passing in the values somehow, or at least passing in the `time.time` function.

Reply:
This is just a log to help us understand if people's blocks are coming late or early, it doesn't do anything

Reply:
Ah got it. Yeah, for logging it's totally fine.